packages/request-manager/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please rename this to `tsconfig.lib.json` and also place here original one `tsconfig.json` from new package template.

`tsconfig.json` - used only for typecheck
`tsconfig.lib.json` - used only for builds

I know it's tempting to combine them because sometimes it can work, but we should do it in same everywhere to prevent confusion and mess in configs.

Reply:
Yes, you are right! Done by fa4468961ba8b899cda2d78f40f18aef79be14a1